main.go

```diff
@@ -197,11 +197,7 @@ type _escFile struct {
 }
 
 func (_escLocalFS) Open(name string) (http.File, error) {
-	f, present := _escData[path.Clean(name)]
-	if !present {
-		return nil, os.ErrNotExist
-	}
-	return os.Open(f.local)
```

Comment:
Test this change yourself with and without using the `-prefix` option. `f.local` is required because there may be a difference between the requested name and the on-disk name. When a prefix is used, the requested name does not contain the prefix, but the on-disk (local) name does.

esc should have an automated test that verifies this, though.